README.md

```diff
@@ -14,29 +14,29 @@ c14bazAAR is an R package to query different openly accessible radiocarbon date
 - [**Installation**](#installation)
 - [**How to use**](#how-to-use) ([Download](#download), [Calibration](#calibration), [Material classification](#material-classification), [Country attribution](#country-attribution), [Duplicates](#duplicates), [Conversion](#conversion), [Technical functions](#technical-functions), [Plotting and visualization](#plotting-radiocarbon-data), [Interaction with other radiocarbon data packages](#other-radiocarbon-packages))
 - [**Databases**](#databases)
-- [**Contributing**](#contributing) ([Adding database getter functions](#adding-database-getter-functions))
+- [**Contributing**](#contributing) ([Adding database getter functions](#adding-database-getter-functions), [Pre-submision testing](#pre-submision-testing), [Versioning](#versioning))
 - [**Citation**](#citation)
 - [**License**](#license)
 
 If you want to use data downloaded with c14bazAAR for your research, you have to cite the source databases. Most databases have a preferred way of citation that also may change over time with new versions and publications. Please check the [respective homepages](#databases) to find out more. The output of c14bazAAR does not contain the full citations of the individual dates, but only a short reference tag. For further information you have to consult the source databases.
 
 ### Installation
 
-We recommend to install the development version from github with the following command (in your R console):
+We recommend to install the stable version from the [R-universe](https://r-universe.dev/) repository of [rOpenSci](https://ropensci.org/) with the following command (in your R console):
 
 ```
-if(!require('remotes')) install.packages('remotes')
-remotes::install_github("ropensci/c14bazAAR")
+install.packages("c14bazAAR", repos = c(ropensci = "https://ropensci.r-universe.dev"))
```

Comment:
Ah - cool. So it works with only this URL.

Reply:
Yes, giving the path to a CRAN mirror is not necessary. At least the package installed for me with only giving the *ropensci.r-universe* path.